src/Ticket.php

```diff
@@ -5121,23 +5121,12 @@ public static function showCentralList($start, $status = "process", bool $showgr
     }
 
     /**
-     * Get tickets count
+     * Get central count criteria
      *
-     * @param boolean $foruser  Only for current login user as requester (false by default)
-     * @param boolean $display  il false return html
-     **/
-    public static function showCentralCount(bool $foruser = false, bool $display = true)
+     * @param boolean $foruser Only for current login user as requester or observer (false by default)
+     */
+    public static function showCentralCountCriteria(bool $foruser = false)
```

Comment:
1. This method is used only privately, so it should be private.
2. `$foruser` is always passed, so I guess its default value can be removed.
3. Please add return type when it can be done without introducing breaking changes.
```suggestion
    private static function showCentralCountCriteria(bool $foruser): array
```